mango-viz/examples/pileup-tutorial.ipynb

```diff
@@ -170,11 +171,49 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": true
    },
+   "source": [
+    "## Genotype Example\n",
+    "\n",
+    "This example shows how to visualize genotypes through a Jupyter widget."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "5958c59ba133449a8407b560c467e02b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# make genotype track\n",
+    "tracks=[Track(viz=\"genotypes\", label=\"my genotypes\", source=pileup.sources.VcfDataSource(\"./data/genodata.v3.vcf\"))]\n",
```

Comment:
I don't see any code modifying the viz genotypes. (see https://github.com/bigdatagenomics/mango/blob/master/mango-viz/bdgenomics/mango/pileup/sources.py#L218 ) Did you forget to push this?

Reply:
Yes. I forgot to push this.. will do that now.